internal/brokers/broker.go

```diff
@@ -17,7 +17,8 @@ import (
 	"golang.org/x/exp/slices"
 )
 
-const localBrokerName = "local"
+// LocalBrokerName is the name of the local broker.
+const LocalBrokerName = "local"
```

Comment:
Maybe  `LocalBrokerId` ?

Since that's how we refer to it in the proto.

Reply:
Definitely